packages/ra-ui-materialui/src/button/Button.tsx

```diff
@@ -54,8 +56,8 @@ export const Button = (props: ButtonProps) => {
                     className={className}
                     color={color}
                     size="large"
-                    {...rest}
```

Comment:
Why did you switch the order?

Reply:
In order to not overlap the value of 'state' prop coming from CreateButton (for example)